packages/cli-tools/src/startServerInNewWindow.ts

```diff
@@ -1,15 +1,15 @@
 import path from 'path';
 import fs from 'fs';
 import execa from 'execa';
-import resolveNodeModuleDir from './resolveNodeModuleDir';
 import logger from './logger';
 import chalk from 'chalk';
+import {findPackageDependencyDir} from './findPackageDependencyDir';
 
 const ERROR = `a dev server manually by running ${chalk.bold(
   'npm start',
 )} or ${chalk.bold('yarn start')} in other terminal window.`;
 
-function startServerInNewWindow(
+async function startServerInNewWindow(
```

Comment:
this doesn't appear to be necessary after the other review comment:
```suggestion
function startServerInNewWindow(
```

Reply:
yeah, good catch 👍